controllers/api/userRoutes.js

```diff
@@ -0,0 +1,108 @@
+const router = require('express').Router();
+const { User, GuestList, Food} = require('../../models');
+
+
+// getting all the users with the guestlist
+router.get('/', (req, res) => {
+    // find all users including its associated guestLists
+    User.findAll({
+        include: [
+            {
+                model: GuestList,
+                attributes: ['id', 'first_name', 'last_name', 'street', 'city', 'state', 'zipcode', 'phone_number', 'email', 'rsvp', 'user_id', 'food_id' ]
```

Comment:
For this route the goal is just to get all of the users, so you can remove that whole include section.  There will be a route with the guestRoutes that will get all the data from the users guest list.  This is what that request will look like: `// getting all the users with the guestlist
router.get("/", (req, res) => {
  User.findAll({
    attributes: { exclude: ["password"] },
  })
    .then((dbUserData) => res.json(dbUserData))
    .catch((err) => {
      console.log(err);
      res.status(500).json(err);
    });
});`

Reply:
Got it